cns/client.go

```diff
@@ -36,6 +36,7 @@ const (
 const (
 	ReleaseVSAN67u3 = "vSAN 6.7U3"
 	ReleaseVSAN70   = "7.0"
+	ReleaseVSAN70u1 = "vSAN 7.0U1"
```

Comment:
Since you are not using this constant in this repo, why do you need this constant? Wouldn't linter fail when you add a var and not use it?

Reply:
I'm planning to use this in a PR in CSI - https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/642/files#diff-f3c4e2561d16ec3070bae674afe9d3afb7c7155fcfa186d8256da34c10f048b1R593
and i thought adding it here maintains consistency. 
AFAIK linter checks dont fail for unused constants that are exported by the package.

Reply:
Fine with me @RaunakShah , thanks.